core/templates/pages/community-dashboard-page/translation-language-select/translation-language-select.component.html

```diff
@@ -0,0 +1,113 @@
+<div #dropdown class="oppia-translation-language-select-container protractor-test-language-selector">
+  <div class="oppia-translation-language-select-inner-container" (click)="toggleDropdown()">
+    {{languageIdToDescription[value]}}
+    <span class="material-icons" style="color: #009688;" [ngStyle]="{ 'transform': dropdownShown ? 'scaleY(-1)' : 'scaleY(1)' }">
+      arrow_drop_down
+    </span>
+  </div>
+
+  <div class="oppia-translation-language-select-featured-tooltip">
+    <span class="oppia-translation-language-select-featured-tooltip-content"
+          *ngIf="descriptionPopupShown"
+          [ngStyle]="{'top': descriptionPopupPxOffsetY + 'px'}">
+      {{descriptionPopupContent}}
+    </span>
+  </div>
+
+  <div *ngIf="dropdownShown" class="oppia-translation-language-select-dropdown-container">
+    <div *ngIf="featuredLanguages.length > 0">
+      <div class="oppia-translation-language-select-dropdown-label">Most needed languages</div>
+      <div *ngFor="let featuredOption of featuredLanguages; index as i">
+        <div class="oppia-translation-language-select-dropdown-option"
+             *ngIf="languageIdToDescription[featuredOption.languageCode]"
+             (click)="selectOption(featuredOption.languageCode)"
+             [ngClass]="{'oppia-translation-language-select-dropdown-option-selected' : featuredOption.languageCode === value}">
+          {{languageIdToDescription[featuredOption.languageCode]}}
+          <span class="material-icons"
+                *ngIf="featuredOption.description"
+                style="color:rgb(183, 183, 183)"
+                (mouseenter)="showDescriptionPopup(i)"
+                (mouseleave)="hideDescriptionPopup()">
+            info
+          </span>
+        </div>
+      </div>
+    </div>
+
+    <div class="oppia-translation-language-select-dropdown-label">All languages</div>
+    <div class="oppia-translation-language-select-dropdown-option"
+         *ngFor="let option of options"
+         (click)="selectOption(option.id)"
+         [ngClass]="{'oppia-translation-language-select-dropdown-option-selected' : option.id === value}"
+    >
+      {{option.description}}
+    </div>
+  </div>
+</div>
+
+<style>
+  .oppia-translation-language-select-container {
+    background-color: #ffffff;
+    border-radius: 4px;
+    box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25);
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 20px;
+    position: relative;
+  }
+
+  .oppia-translation-language-select-inner-container {
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+  }
+
+  .oppia-translation-language-select-dropdown-container {
+    background-color: #ffffff;
+    border-radius: 4px;
+    box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25);
```

Comment:
Remove px from 0 (ditto elsewhere in this file

Reply:
Done